OpenSim/Simulation/Model/Frame.cpp

```diff
@@ -180,12 +180,42 @@ SimTK::Vec3 Frame::expressVectorInAnotherFrame(const SimTK::State& state,
     return findTransformBetween(state, frame).R()*vec;
 }
 
-SimTK::Vec3 Frame::findLocationInAnotherFrame(const SimTK::State& state, const
-        SimTK::Vec3& point, const Frame& otherFrame) const
+SimTK::Vec3 Frame::expressVectorInGround(const SimTK::State& state,
+                                const SimTK::Vec3& vec_F) const
+{
+    return getTransformInGround(state).R()*vec_F;
```

Comment:
Oh, sorry I hadn't noticed this distinction. It smelled like L186 might have been doing some unnecessary work, but I smell the difference now that you've defined `express` and `find`. I suggest adding `express` to the [list of naming conventions in CONTRIBUTING.md](https://github.com/opensim-org/opensim-core/blob/master/CONTRIBUTING.md#naming-conventions).